.github/workflows/integration.yml

```diff
@@ -43,9 +47,7 @@ on:
         description: 'Kubernetes version'
         type: choice
         options:
-          - "['1.23']"
           - "['1.24']"
-          - "['1.25']"
```

Comment:
We need  options:
 - "['1.25']"
 - "['1.26']"

Reply:
I'll add them